src/peerGroup.js

```diff
@@ -1,7 +1,7 @@
 'use strict'
 
 var debug = require('debug')('bitcoin-net:peergroup')
-var dns = require('dns')
+try { var dns = require('dns') } catch (err) {}
```

Comment:
This also shouldn't be necessary since browserify has a default definition for the `dns` module (an empty object).